data/data/vsphere/master/ignition.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  ignition_encoded = "data:text/plain;charset=utf-8;base64,${base64encode(var.ignition)}"
+}
+
+data "ignition_file" "hostname" {
```

Comment:
if the machine-api is to recreate the machine, how will it recreate this behavior?